tests/cypress/tests-e2e/legacyParamImport.cy.js

```diff
@@ -324,6 +341,18 @@ describe('Test on legacy param import', () => {
                 expect(query.toString()).to.equal(expectedQuery.toString())
             })
         })
+        it('Sets the background layer correctly', () => {
+            cy.goToMapView(
+                {
+                    bgLayer: 'test.background.layer2',
+                },
+                false
+            )
+            cy.readStoreValue('state.layers.currentBackgroundLayer').then((bgLayer) => {
+                expect(bgLayer).to.not.be.null
+                expect(bgLayer.id).to.eq('test.background.layer2')
+            })
+        })
```

Comment:
bglayer is not a legacy parameter and we should not test it here, it is already tested in layers.cy.js

Reply:
bgLayer is part of the official parameters of the old viewer. I figured I would also test it through a legacy test to make sure it works well. If you feel this is not useful, I'll remove it.

Reply:
That being said, there is already a tests which checks that params that do not change are transferred as is. So we can indeed ignore the official old parameters which remained the same